tests/integration/test_local_variables.py

```diff
@@ -573,31 +573,33 @@ def test_trashed_locals(generate_threads, python, tmpdir):
 
 class ListObject(ctypes.Structure):
     _fields_ = [
-        ("ob_refcnt", ctypes.c_ssize_t),
         ("ob_type", ctypes.c_void_p),
         ("ob_size", ctypes.c_ssize_t),
         ("ob_item", ctypes.c_void_p),
     ]
 
 class TupleObject(ctypes.Structure):
     _fields_ = [
-        ("ob_refcnt", ctypes.c_ssize_t),
         ("ob_type", ctypes.c_void_p),
         ("ob_size", ctypes.c_ssize_t),
         ("ob_item0", ctypes.c_void_p),
         ("ob_item1", ctypes.c_void_p),
     ]
 
+def ob_type_field(obj):
```

Comment:
Is this something we can avoid with the new offsets we added to 3.13?

Reply:
It's technically possible, but would require much more complexity. We'd need a way for pystack to attach to a running process and dump the offsets rather than the stack, and we'd need to then take those offsets and pass them to program being run by the test.

We've got options if this turns out to be a maintenance burden, but for now, I'd rather keep this as simple as we can.